frontend/packages/console-dynamic-plugin-sdk/README.md

```diff
@@ -31,6 +31,18 @@ provided by Console to its dynamic plugins.
 _[OpenShift Console Dynamic Plugins feature page][console-doc-feature-page]_ - A high-level overview of
 dynamic plugins in relation to OLM operators and cluster administration.
 
+## Plugin project references
+
+If you're new to dynamic plugins, we suggest to clone the [Console plugin template][console-plugin-template]
+repo and try to build and run it using a [local Console Bridge server](#local-plugin-development).
```

Comment:
Typically we recommend running console in a container runtime to avoid needing to build locally. (See the start console script in the template repo.)